src/story.ts

```diff
@@ -1,43 +1,127 @@
+import { SBType } from './SBType';
+
 export type StoryId = string;
-export type StoryKind = string;
+export type ComponentId = string; // TODO -- should this be in storyId?
```

Comment:
By this I mean, should we guarantee `ComponentId` is always set in the `StoryIdentifier` type (which forms the basis of all "contexts")